otx/cli/tools/export.py

```diff
@@ -90,7 +95,9 @@ def main():
 
     task.export(ExportType.OPENVINO, exported_model)
 
-    os.makedirs(args.save_model_to, exist_ok=True)
+    if "save_model_to" not in args or not args.save_model_to:
+        args.save_model_to = str(config_manager.workspace_root / "model-exported")
+    Path(args.save_model_to).mkdir(exist_ok=True)
```

Comment:
```suggestion
    Path(args.save_model_to).mkdir(exist_ok=True, parents=True)
```
Isn't it needed?

Reply:
reflected this. https://github.com/openvinotoolkit/training_extensions/pull/1641/commits/3fcf067f3ae5f70826b0df66bdb702db0a04c358

Reply:
It allows the user can input the more complex path (e.g.`results/checkpoints`), not mandatory.